lib/project/result-view.coffee

```diff
@@ -44,11 +44,61 @@ class ResultView extends View
       @hide()
     else
       @show()
+      @addContextToMatches(@filePath, matches)
       for match in matches
         @matches.append(new MatchView(@model, {@filePath, match}))
 
     @matches.children().eq(selectedIndex).addClass('selected') if selectedIndex > -1
 
+  addContextToMatches: (filePath, matches) ->
+    CONTEXT_LINES = atom.config.get('find-and-replace.searchContextExtraLines')
+    return if CONTEXT_LINES is 0
+    content = fs.readFileSync(filePath).toString()
+    lines = content.split('\n')
+    prevMatch = match
+    prevRowIndex = 0
+    prevLinesAfter = 0
+    for matchIndex in [0...matches.length]
+      match = matches[matchIndex]
+      range = match.range
+      if not range
```

Comment:
Fixed in https://github.com/atom/find-and-replace/pull/769/commits/9fe4836a303924e60dfbacbe6ba2090de1f4c0af